src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.scss

```diff
@@ -16,6 +16,10 @@
     right: 2rem;
     top: 2rem;
   }
+
+  .custom-svg {
+    margin-top: 5px;
```

Comment:
Can we use carbon based [spacings](https://carbondesignsystem.com/elements/spacing/overview/#spacing-scale) ?